babel.config.js

```diff
@@ -11,10 +13,13 @@ module.exports = function (api) {
         // modules: false
       },
     ],
-    '@babel/preset-react',
   ];
 
-  const plugins = ['@babel/plugin-transform-object-assign', 'inline-react-svg'];
+  const plugins = [
+    '@babel/plugin-proposal-class-properties',
```

Comment:
Updates our default supported plugins to use `@babel/plugin-proposal-class-properties` and `@babel/plugin-proposal-object-rest-spread`